Owl/app/src/main/java/com/example/owl/ui/courses/FeaturedCourses.kt

```diff
@@ -78,10 +82,15 @@ fun FeaturedCourse(
         elevation = OwlTheme.elevations.card,
         shape = MaterialTheme.shapes.medium
     ) {
+        val featuredString = stringResource(id = R.string.featured)
         ConstraintLayout(
-            modifier = Modifier.clickable(
-                onClick = { selectCourse(course.id) }
-            )
+            modifier = Modifier
+                .clickable(
+                    onClick = { selectCourse(course.id) }
+                )
+                .semantics {
+                    accessibilityLabel = featuredString
```

Comment:
By explicitly setting the accessibility label, you are overriding the accessibility label that's generated for this constraint layout by default. Is that your intention? If you only want to have a reference to this in your test, you can use the testTag property instead.